envier/env.py

```diff
@@ -228,7 +268,18 @@ class Env(object):
 
     def __init__(self, source=None, parent=None):
         # type: (Optional[Dict[str, str]], Optional[Env]) -> None
-        self.source = source or os.environ
+        # Has to come first to avoid issues with __setattr__
+        self._items = defaultdict(
+            lambda: _EnvSource(
+                [
+                    "programmatic",
```

Comment:
```suggestion
                    "override",
```
?

Reply:
yeah I wasn't a fan of `programmatic`, `override` is better :+1:

Reply:
With remote configuration being a thing I opted for `code` to reflect in-code changes to the configuration rather than `override`